### PR TITLE
Move Existing Tag related messages to Settings Snippet Toggles

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue:

- #5143 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/tagmanager/components/common/FormInstructions.js [hunk 1]
```diff
@@ -110,37 +98,19 @@ export default function FormInstructions( { showExistingTagMessage } ) {
 		return (
 			<p>
 				{ __(
-					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below. You can change these later in your settings.',
```
Comment (eugene-manuilov) [inline assets/js/modules/tagmanager/components/common/FormInstructions.js:113]: @kuasha420 why do we need to remove this text? I don't see it neither in AC nor IB.

File: assets/js/modules/tagmanager/components/common/FormInstructions.js [hunk 2]
```diff
@@ -110,37 +98,19 @@ export default function FormInstructions( { showExistingTagMessage } ) {
...
-						__(
-							'An existing tag was found on your site (%s). If you later decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first.',
-							'google-site-kit'
-						),
-						containerID
-					) }
-				</p>
+		<p>
+			{ __(
+				'Please select your Tag Manager account and container below.',
```
Comment (eugene-manuilov) [inline assets/js/modules/tagmanager/components/common/FormInstructions.js:111]: Same here. Shouldn't it be `Please select your Tag Manager account and container below. You can change these later in your settings.`?

File: assets/js/modules/analytics/components/common/SetupUseSnippetSwitch.js [hunk 3]
```diff
@@ -79,5 +79,5 @@ export default function SetupUseSnippetSwitch() {
 			</Fragment>
 		);
 
-	return <UseSnippetSwitch description={ description } isSetup />;
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics/components/common/SetupUseSnippetSwitch.js:82]: Same here, remove the `isSetup` property from the `UseSnippetSwitch` component: https://github.com/google/site-kit-wp/blob/5bdc7c059312f18a1464794b3f8a69bf0e5319bb/assets/js/modules/analytics/components/common/UseSnippetSwitch.js#L41

See in IB:
> * In UseSnippetSwitch components of **analytics**, analytics-4 and tagmanager modules:
>   * Remove isSetup prop and the class that it adds conditionally.
>   * Remove unused imports.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @kuasha420. Almost looks good to me. Added a few comments. Please, take a look.

File: assets/js/modules/tagmanager/components/common/FormInstructions.js (hunk 1 above)
Comment (kuasha420) [reply assets/js/modules/tagmanager/components/common/FormInstructions.js:113]: I didn't realize that AC doesn't cover it but this was an issue spotted by @wpdarren on #5046 where it was saying "You can later change it in the settings" in the Settings page as well. I assumed that the FormInstruction was only used in Settings but It's also used in SEtup. I had made adjustment to it so if it's Setup, only then the `You can change these later in your settings` is shown. Does it look reasonable to you? Cheers.

File: assets/js/modules/tagmanager/components/common/FormInstructions.js (hunk 2 above)
Comment (kuasha420) [reply assets/js/modules/tagmanager/components/common/FormInstructions.js]: See above comment

File: assets/js/modules/tagmanager/components/common/FormInstructions.js (hunk 1 above)
Comment (eugene-manuilov) [reply assets/js/modules/tagmanager/components/common/FormInstructions.js:113]: Yes, thanks a lot, @kuasha420.

File: assets/js/modules/tagmanager/components/common/FormInstructions.js (hunk 2 above)
Comment (eugene-manuilov) [reply assets/js/modules/tagmanager/components/common/FormInstructions.js]: Yep, works for me.

**Review by eugene-manuilov — APPROVED** [review]
LGTM. Thanks, @kuasha420.

**Pull request merged**